src/main/ui/chat/chat-ui.gd

```diff
@@ -143,10 +135,13 @@ func _increment_line() -> bool:
 			_rewinding_text = false
 	
 	if not _rewinding_text:
-		if _chat_event_index + 1 < _chat_events.size():
-			# continue forward through chat events
-			_chat_event_index += 1
-			did_increment = true
+		var link_index := 0
+		var links: Array = _chat_tree.get_event().links
+		if links and links.size() >= 2:
+			print("Choose one: %s" % [_chat_tree.get_event().link_texts])
+			link_index = clamp(1, 0, links.size() - 1)
+			print("  (%s)" % _chat_tree.get_event().link_texts[link_index])
+		did_increment = _chat_tree.advance(link_index)
```

Comment:
Split this into two branches for clarity: the 'navigating the default branch' branch, and the 'continuing' branch

Reply:
Split into three explicit branches instead of the two which were there before.